entrypoint.sh

```diff
@@ -44,6 +44,7 @@ elif [ -n "$POSTGRES_PORT_5432_TCP" ]; then
 		echo >&2 "  Connecting to DB_HOST ($DB_HOST)"
 		echo >&2 "  instead of the linked postgres container."
 	fi
+	: ${DB_NAME:='postgres'}
```

Comment:
yes, and by using the defaults, I can specify a `docker-compose.yml` for postgres nearly identical to mysql, with only the:

- database image name
- link name
- database password environment variable

changed. If you don't allow this, then the burden is on the user to lookup the way to create a new database in postgres. I don't think that make sense.

If I don't need to look up how to create a database for the mysql version to work, why should I have to do that for postgres?

Lastly, please note that creating a database as part of the postgres boot process *isn't documented* on the postgres image page. See [the image's readme](https://registry.hub.docker.com/_/postgres/) and the [entrypoint.sh](https://github.com/docker-library/postgres/blob/master/9.3/docker-entrypoint.sh#L45)